ctapipe/tools/make_irf.py

```diff
@@ -102,7 +102,7 @@ def get_sim_info_and_spectrum(self, loader):
     def setup(self):
         self.tc = ToolConfig(parent=self)
```

Comment:
Having a separate class for the config is not needed, you can just add the traits directly to the tool itself.